projects/common-library-typescript/build.sh

```diff
@@ -1,3 +1,4 @@
 npm link ../architecture
+npm install --unsafe-perm grpc@1.8.4
```

Comment:
Can you add a comment of why it was required, so that we won't forget?

Reply:
Done.